app/frontend/src/App.tsx

```diff
@@ -15,20 +15,17 @@ import AuthProvider from "./features/auth/AuthProvider";
 import { ReactQueryClientProvider } from "./reactQueryClient";
 import { theme } from "./theme";
 
-const useStyles = makeStyles((theme) => ({
+const useStyles = makeStyles({
   padding: {
-    paddingBottom: theme.spacing(7),
+    paddingBottom: theme.shape.navPaddingMobile,
     paddingLeft: theme.spacing(2),
     paddingRight: theme.spacing(2),
-    [`${theme.breakpoints.up("xs")} and (orientation: landscape)`]: {
-      paddingBottom: theme.spacing(6),
-    },
```

Comment:
Am I correct to get rid of this? I don't see why landscape needs special treatment. Either the breakpoint is larger than md, and it uses the desktop padding, or it isn't, so it uses the mobile padding.

Reply:
Yeah I'm not sure what this was trying to achieve... let's remove it for now to simplify the CSS and we can always fix it if people report visual bugs